dips/dip-158.md

```diff
@@ -0,0 +1,794 @@
+---
+dip: 158
+title: P2M (eCommerce) Payments Off-chain API
+authors: Daniel Prinz, Yinon Zohar, Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 03/15/2021
+---
+
+# Summary
+
+This DIP is an extension of the Off-Chain protocol for enabling eCommerce P2M payments (Peer To Merchant) of the following types:
+* Charge
+* Auth / Capture
+ 
+This DIP extends [DIP-1](https://dip.diem.com/dip-1/) which specifies P2P payment flows (Peer To Peer).
+
+This DIP replaces the Auth/Capture section in [DIP-8](https://dip.diem.com/dip-8/#authcapture).
+
+This DIP does NOT include payment flows which require funds pull pre-approval such as subscription and recurring payments. 
+
+This DIP makes use of the `result` field added to the `CommandResponseObject` as described in DIP-10
+
+---
+# Abstract / Motivation
+
+
+This DIP, as described here, is an extension of the Off-Chain Protocol to support more advanced payment functionality - particularly eCommerce P2M payments. 
+
+## How P2M Payments Different than P2P Transfers 
+There are some key differences in the process of paying a merchant (P2M) that differ from simple P2P transfer:
+
+|      |P2M|P2P|
+|------|---|---|
+|**Payer Data**|Buyer details are required for **every** payment / purchase in order to perform risk checks and apply AML policies|Required only for transfers for which Travel Rule applies. i.e., over 1000 USD|
+|**Payment Owner**|The merchant, i.e. the receiver of funds, "drives" the process and controls the type of payment and its timing. For example, the merchant will decide when to capture the funds and whether to capture the entire amount or only partial amount |The payer, i.e. the sender of funds, determines the nature of the transfer|
+|**Reconciliation**|The merchant requires a way to correlate the transfer of funds to a payment or payment request for reconciliation purposes|No reconciliation|
+|
+
+## P2M Payment Types
+There are two types of P2M payments described in this document: 
+
+1. **Charge**. Charge payment stipulates an immediate capture of funds. This means that a successful off-chain negotiation for such payment will result in funds being captured 
+2. **Auth / Capture**. Auth / Capture payment, as the name suggests, means that the payment is divided to two steps: Authorization and Capture. Successful Authorization implies that the funds are locked so they can later be captured
+
+This DIP defines the type of eCommerce P2M payments, corresponding requests and the payment structures. 
```

Comment:
I feel that this sentence is orphan -- what does it refer to?

Reply:
Correct. I will move it to the summary section